README.md

```diff
@@ -8,7 +8,7 @@ Example:
 GET
 ```java
 HttpRequest request = httpRequest(new TargetUrl("abv.bg")).build();
-ChitaHttp chitaClient = new ChitaHttpClient();// in most cases chitaClient should be injected
+ChitaHttp chitaClient = new HttpClient();// in most cases chitaClient should be injected
```

Comment:
Not sure what ChitaHttp interface gives you. 

I don't think that we should relay on mocking of the http client.